Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
```

Comment:
I'd rather not `bin.install Dir["bin/*]`, since this would clutter up `/usr/local/bin` with links to a lot of other junk, not just the program binaries. Even so, I've only linked to a selection of the binaries; the Debian package (based on v1.2 from years ago) actually installs [a lot more of them](https://packages.debian.org/wheezy/amd64/speech-tools/filelist).